test/integration/test_create_dss_docker_image.py

```diff
@@ -16,6 +18,22 @@
 from exasol.ds.sandbox.lib import pretty_print
 
 
+class StderrCapture:
+    def __init__(self):
+        self._stream = sys.stdout
+        self._buffer = io.StringIO("")
+        sys.stdout = self._buffer
+
+    def stop(self):
+        sys.stdout = self._stream
+
+    @property
+    def output(self) -> str:
+        """Return the captured output."""
+        self._buffer.seek(0)
+        return self._buffer.read().strip()
```

Comment:
where is `sys.stderr`? I only see `sys.stdout` the class name states it is `StderrCapture` though.
The context lib may be of help here [redirect_stdout](https://docs.python.org/3/library/contextlib.html?highlight=context%20manager#contextlib.redirect_stdout)